app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Hi you did a great job but as part of the requirements, you are supposed to ask for confirmation so ill recommend adding `:confirmable` to your device

Reply:
Thank you @m05e5 for reviewing my project 🙏